src/helpers.js

```diff
@@ -111,7 +155,15 @@ const formStateToCustomizations = (customizations) => {
     services,
   };
 
-  return customizationsParsed;
+  // if a user steps through each component in the form without filling any
+  // of the steps out, there is a possibility that each form field is empty.
+  // if this is the case, we want to return an empty (or undefined) `customizationsParsed`
+  // object
+  const everythingUndefined = Object.keys(customizationsParsed).every(
```

Comment:
I could simplify this by only initialising the `customizationsParsed` if a field is not empty... but this was a quick win